### ISSUE TITLE
torch not compiled

### ISSUE THREAD

**Issue by naveens01** [microsoft/visual-chatgpt#80]
torch not complied with cuda

**Comment by wrk226** [comment]
conda install pytorch==1.12.1 torchvision==0.13.1 cudatoolkit=11.6 -c pytorch -c conda-forge

**Comment by SingularityMan** [comment]
> conda install pytorch==1.12.1 torchvision==0.13.1 cudatoolkit=11.6 -c pytorch -c conda-forge

Already Tried this, Same assertion error:

AssertionError: Torch not compiled with CUDA enabled

**Comment by alinemati45** [comment]
same here! for Macbookpro

**Comment by SingularityMan** [comment]
> Can i use cpu instead of gpu?

I don't think so. When you compile pytorch without CUDA it raises the Assertion Error.

**Comment by Marysa73** [comment]
ERROR: Could not find a version that satisfies the requirement cudatoolkit~=11.3 (from controlnet) (from versions: none)
ERROR: No matching distribution found for cudatoolkit~=11.3
